documentation/modules/con-user-model.adoc

```diff
@@ -1,11 +1,12 @@
 // Module included in the following assemblies:
 //
 // assembly-tenant-guide.adoc
+// master.adoc
 
 [id='con-user-model-{context}']
 = User model
 
-You can create users to access messaging clients and the {ProductName} Console through user definition files, which specify the authentication type and authorization policies.
+You can create users to access messaging clients through user definition files, which specify the authentication type and authorization policies.
```

Comment:
Not relating to this change, but "create users to access messaging clients" is not clear.  Messaging clients (applications) connect using a messaging user,

Reply:
@k-wall Thanks for pointing this out. Will fix this.